libs/common/src/constants/portal/activity.ts

```diff
@@ -4,10 +4,12 @@ export const PORTAL_ACTIVITY_LABEL = {
   AIN_SUBMISSION: 'Automatic inclusion notice submitted to UKEF',
   FACILITY_CHANGED_ISSUED: 'Bank facility stage changed',
   DEAL_CANCELLED: 'Deal cancelled',
+  DEAL_CANCELLATION_SCHEDULED: 'Deal cancellation scheduled',
 } as const;
 
 export const PORTAL_ACTIVITY_TYPE = {
   DEAL_CANCELLED: 'DEAL_CANCELLED',
+  DEAL_CANCELLATION_SCHEDULED: 'DEAL_CANCELLATION_SCHEDULED',
```

Comment:
Is `scheduled` from the designs - I thought we'd moved to using the word `Pending`

Reply:
Good shout - I have asked design. Currently I can only see some screenshots in the ticket.

Reply:
Confirmed with design, it should be "pending". Updated.